cmd/oras/internal/option/format.go

```diff
@@ -68,6 +68,11 @@ var (
 		Name:  "text",
 		Usage: "Print in text format",
 	}
+
+	FormatTypeTreeFull = &FormatType{
+		Name:  "tree-full",
```

Comment:
I'm not in love with the name `tree-full`, but I don't have a better idea.

Reply:
I do like the idea of showing the annotations in the normal tree view.

Reply:
Sorry for changes in direction on this @Horiodino

Reply:
Let's continue the discussion in https://github.com/oras-project/oras/issues/1534. We will temporarily hold on this PR until we aligned the user experience. Apologies for blocking you @Horiodino